rfc/rfc002-authentication-token.md

```diff
@@ -288,11 +275,29 @@ The response is quite lengthy and contains tons of information. Luckily the IRMA
 
 The response is used as part of the authorization token in the [OAuth flow](rfc003-oauth2-authorization.md).
 
-#### 7.1.4 AuthenticationToken \_\_Container usage
+#### 7.1.4 Verifiable Presentation
+
+Below is a complete example of how an Irma proof can be embedded in a VP:
+
+```yaml
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": ["VerifiablePresentation", "NutsIrmaPresentation"],
+  "proof": {
+    "type": "NutsIrmaSignedContract",
+    "proofValue": "DgYdYMUYHURJLD7xdnWRinqWCEY5u5fK...j915Lt3hMzLHoPiPQ9sSVfRrs1D"
+  }
+}
+```
 
-To use the IRMA _Authentication Token_ in a Authentication_Token Container_, the `type` member of the Authentication_Token Container_ MUST set to the value `irma`
+Beside the mandatory VP fields, the following applies:
 
-The `token` value MUST be the base64 encoded JSON response.
+* The `type` field MUST be set to the value `["VerifiablePresentation", "NutsIrmaPresentation"]`.
```

Comment:
does order matter? Is the VP allowed to have more types than just those 2?

If so, could simplify it to "type must contain at least VerifiablePresentation and NutsIrmaPresentation", otherwise it should be validated by implementations that its exactly as spec-ed here (leading to more code).

Reply:
changed to 'MUST contain the following values'